tic-tac-toe-app/GameBoard.jsx

```diff
@@ -0,0 +1,69 @@
+function GameBoard() {
+  const [turn, setTurn] = React.useState(1);
+  const [cells, setCells] = React.useState([
+    null, null, null,
+    null, null, null,
+    null, null, null,
+  ]);
+
+  const checkWinner = () => {
+    // turn >= 5 && turn !== 9
+    if (true) {
+      // horizontal wins
```

Comment:
It appears that `true` will always be so and that the code block will always be executed. If that's the case, this if statement on line 11 does not seem needed.